docs/command/mongocli-config-set.txt

```diff
@@ -15,7 +15,7 @@ mongocli config set
 Configure specific properties of a profile.
 
 Configure specific properties of the profile.
-Available properties include: [project_id org_id service public_api_key private_api_key output ops_manager_url base_url ops_manager_ca_certificate ops_manager_skip_verify mongosh_path skip_update_check].
+Available properties include: [project_id org_id service public_api_key private_api_key output ops_manager_url base_url ops_manager_ca_certificate ops_manager_skip_verify mongosh_path skip_update_check access_token refresh_token].
```

Comment:
FYI @mongodb/docs-cloud-team we are just adding two new props here, examples and other docs are being handled by this other PR https://github.com/mongodb/mongocli/pull/833